docs/document/content/others/api-change-history/shardingsphere-jdbc/yaml-configuration.cn.md

```diff
@@ -1,9 +1,9 @@
 +++
 title = "YAML 配置"
-weight = 2
+weight = 1
 +++
 
-## 5.0.0-alpha
+## ShardingSphere-5.0.0-beta
```

Comment:
If 5.0.0-beta is different from alpha, a new part should be added for beta.

Reply:
Hi, #10981 refers to ShardingSphere-5.0.0-beta, do I need to roll back to 5.0.0-alpha.